tests/Paramore.Brighter.Core.Tests/CommandProcessors/TestDoubles/SpyTransaction.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Paramore.Brighter.Core.Tests.CommandProcessors.TestDoubles;
+
+public class SpyTransaction
```

Comment:
Similar to in-memory, does this imply that perhaps we need to swap from CommittableTransacton to our own InMemoryTransaction to use?